assembl/graphql/idea.py

```diff
@@ -760,27 +663,59 @@ def mutate(root, args, context, info):
 
                 video_media = video.get('media_file', None)
 
-            # Our thematic, because it inherits from Idea, needs to be
-            # associated to the root idea of the discussion.
-            # We create a hidden root thematic, corresponding to the
-            # `identifier` phase, child of the root idea,
-            # and add our thematic as a child of this root thematic.
-            root_thematic = get_root_thematic_for_phase(discussion, identifier)
-            if root_thematic is None:
-                root_thematic = create_root_thematic(discussion, identifier)
+            parent_idea_id = args.get('parent_id')
+            if parent_idea_id:
+                parent_idea_id = int(
+                    Node.from_global_id(parent_idea_id)[1])
+                if parent_idea_id:
```

Comment:
You don't need to test parent_idea_id here. int() or from_global_id raise an error if parent_idea_id is invalid .

Reply:
yep